exporter/coralogixexporter/README.md

```diff
@@ -50,7 +50,7 @@ Prometheus is currently the leading tool for metric collection, it’s easy to i
 exporters:
   prometheusremotewrite:
     # The Coralogix metrics ingress endpoint
-    endpoint: "https://api.coralogix.com"
+    endpoint: "https://metrics-api.coralogix.com"
```

Comment:
Interesting, the tracing endpoint doesn't have protocol, but the metrics does?

Reply:
Yes, metrics endpoint has "https" protocol but to traces endpoint has no protocol

Reply:
Thanks for confirming.